projects/notary/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder-go
+RUN git clone --depth 1 https://github.com/notaryproject/notary
```

Comment:
`https://github.com/notaryproject/notary` is for `notary v1` only.

For notary v2, the repo paths are as follows:

- CLI: `https://github.com/notaryproject/notation`
- Lib: `https://github.com/notaryproject/notation-go`
- Lib: `https://github.com/notaryproject/notation-core-go`